encoding/dot/dotparser/testdata/.gitignore

```diff
@@ -0,0 +1,5 @@
+*.dot
```

Comment:
There is a reason for putting it there, specifically in the `testdata` directory. And that is that the Makefile downloads the entire test suite of Graphviz, and converts a huge number of DOT files, tracking differences between the Graphviz implementation and the Go DOT parser. The intention is not to include original Graphviz test cases into this repo, but rather to use `make fetch` to download them.